### PR TITLE
Select reviewees

### PR DESCRIPTION
**Pull request by Snappiestjack**
With this commit it is now possible to select other users that can view/review the image.

Backend only. Ignore the frontend! This is to be done.

### CODE REVIEW

**Comment by kerrylewis** [comment]
@Snappiestjack the system doesn't seem to work if you're the only registered user. 
image

**Comment by Snappiestjack** [comment]
@kerrylewis See latest commit

**Review by kerrylewis — APPROVED** [review]

File: .gitignore [hunk 1]
```diff
@@ -28,6 +28,7 @@ bld/
 .vs/
 # Uncomment if you have tasks that create the project's static files in wwwroot
 #wwwroot/
+Migrations/
```
Comment (wilka) [inline .gitignore:31]: Ignoring the `Migrations` folder looks like it was a mistake here. Important stuff goes in that folder.

Comment (Snappiestjack) [reply .gitignore:31]: We find it easier to delete and rebuild the database from scratch when changing between different pull requests.

Comment (wilka) [reply .gitignore:31]: Ah, Ok. That won't work once there's a live DB though (i.e. migrations will need to be sorted), but it'll do for dev-only testing at the moment.

Comment (Snappiestjack) [reply .gitignore:31]: Fair. It's just that when we'r working on different things one of us may need a more up to date DB than someone else.

**Pull request merged**